.github/actions/multi-approvers/tests/main.test.ts

```diff
@@ -62,4 +87,30 @@ test("#main", { concurrency: true }, async (suite) => {
       "Multi-approvers action failed: unexpected event [push].",
     );
   });
+
+  await suite.test(
+    "should fail if user-id-allowlist contains invalid numeric ID",
+    async (t) => {
+      const invalidAllowlistInput = "123,abc,456";
+      const core = newFakeCore({
+        token: "fake-token",
+        team: "fake-team",
+        "user-id-allowlist": invalidAllowlistInput,
+      });
+      const setFailed = t.mock.method(core, "setFailed", () => {});
+      const context = createFakePullRequestContext();
+
+      await main(core, context);
+
+      assert.equal(setFailed.mock.calls.length, 1);
+      const failMsg = String(setFailed.mock.calls[0].arguments[0]);
+
+      assert.ok(
+        failMsg.includes(
```

Comment:
If at all possible, we should be as explicit as possible in our code. Therefore, we should use assert.equal instead of only partially checking the failure message.